packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx

```diff
@@ -87,6 +87,13 @@ export const MetricsTable = () => {
         fetchMoreOnBottomReached(tableContainerRef.current);
     }, [fetchMoreOnBottomReached]);
 
+    const totalResults = useMemo(() => {
+        if (!data) return 0;
+        // Return total results from the last page, this should be the same but still we want to have the latest value
+        const lastPage = data.pages[data.pages.length - 1];
+        return lastPage.pagination?.totalResults ?? 0;
+    }, [data]);
```

Comment:
isn't `flatData` sufficient? or is there more to consider I'm missing?

Reply:
`flatData` will only include the results that have been fetched
`totalResults` on the other hand has the full count of the query when not paginated which is the total number of metrics

Reply:
ahhh I see! ty